apps/condo/domains/user/schema/RegisterNewUserService.js

```diff
@@ -99,28 +100,31 @@ const RegisterNewUserService = new GQLCustomSchema('RegisterNewUserService', {
             },
             resolver: async (parent, args, context) => {
                 const { data } = args
-                const { confirmPhoneActionToken, ...restUserData } = data
+                // TODO(DOMA-3209): check dv! and make it required
+                const { confirmPhoneActionToken, dv, phone, email, ...restUserData } = data
                 const userData = {
                     ...restUserData,
+                    email: normalizeEmail(email),
+                    phone: normalizePhone(phone),
```

Comment:
AFAIK, normalizePhone will return null in case of wrong format. Maybe to validate here explicitly?

Reply:
yes, bun not in that PR